templates/tweet/index.html

```diff
@@ -1 +1,172 @@
+                                By signing up, you agree to the Terms of Service and Privacy Policy, including Cookie Use.
+                            </div>
+                            <br>
+                            <h5>Already have an account?</h5>
+                            <a href="javascript.void(0)" data-bs-toggle="modal" data-bs-target="#sign_in_modal" class="btn_1 full-width">Sign In</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <!-- /row -->
+        </div>
+
+
+
+
+
+        <!-- Sign Up Modal -->
+		<div class="modal fade" id="sign_up_modal">
+			<div class="modal-dialog">
+				<div class="modal-content">
+					<div class="modal-header">
+						<h3>Sign Up</h3>
+						<button type="button" class="btn-close" data-bs-dismiss="modal"></button>
+					</div>
+					<div class="modal-body">
+						<form id="sign-in-dialog" method="post" action="{% url 'user:Signup' %}">
+							{% csrf_token %}
+							<div class="sign-in-wrapper">
+								<div class="form-group">
+									<label>Name</label>
+									<input type="text" class="form-control" name="name" required>
+								</div>
+								<div class="form-group">
+									<label>Email</label>
+									<input type="email" class="form-control" name="email" required>
+								</div>
+
+								<div class="form-group">
+									<label>Password</label>
+									<input type="password" class="form-control" name="password" required>
+								</div>
+
+								<div class="text-center">
+									<input type="submit" value="Sign Up" class="btn_1 full-width">
+									Already have an account? <a href="javascript.void(0)" data-bs-toggle="modal" data-bs-target="#sign_in_modal">Sign In</a>
+								</div>
+							</div>
+						</form>
+					</div>
+				</div>
+			</div>
+		</div>
+		<!-- /Sign Up Modal -->
+
+
+
+
+		<!-- Sign In Modal -->
+		<div class="modal fade" id="sign_in_modal">
+			<div class="modal-dialog">
+				<div class="modal-content">
+					<div class="modal-header">
+						<h3>Sign In</h3>
+						<button type="button" class="btn-close" data-bs-dismiss="modal"></button>
+					</div>
+					<div class="modal-body">
+
+						<form id="sign-in-dialog" method="post" action="{% url 'user:Signin' %}">
+							{% csrf_token %}
+							<div class="sign-in-wrapper">
+								<div class="form-group">
+									<label>Email / Username / Phone</label>
+									<input type="text" class="form-control" name="email">
+								</div>
+								<div class="form-group">
+									<label>Password</label>
+									<input type="password" class="form-control" name="password">
+								</div>
+								<div class="clearfix add_bottom_15">
+									<div class="float-end mt-1">
+										<a id="forgot" href="javascript:void(0);" data-bs-toggle="modal" data-bs-target="#forgot_password_modal">
+											Forgot Password?
+										</a>
```

Comment:
remove this if we don't have functionallity yet

Reply:
deleted